diff --git a/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java b/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java
index 6a3297f..951786c 100644
--- a/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java
+++ b/node_modules/react-native-sms-retriever/android/src/main/java/me/furtado/smsretriever/PhoneNumberHelper.java
@@ -1,5 +1,6 @@
 package me.furtado.smsretriever;
 
+import java.util.Random;
 import android.app.Activity;
 import android.app.PendingIntent;
 import android.content.Context;
@@ -109,7 +110,8 @@ final class PhoneNumberHelper {
 
             if (activity instanceof FragmentActivity) {
                 final FragmentActivity fragmentActivity = (FragmentActivity) activity;
-                builder = builder.enableAutoManage(fragmentActivity, mApiClientOnConnectionFailedListener);
+                Random rand = new Random();
+                builder = builder.enableAutoManage(fragmentActivity, rand.nextInt(1000000000), mApiClientOnConnectionFailedListener);
             }
 
             mGoogleApiClient = builder.build();
