diff --git a/node_modules/react-native-set-version/build/index.js b/node_modules/react-native-set-version/build/index.js
index 60baf53..5fde60b 100755
--- a/node_modules/react-native-set-version/build/index.js
+++ b/node_modules/react-native-set-version/build/index.js
@@ -173,7 +173,7 @@ const changeVersion = async () => {
   const appName = setPackageVersion(versionText).name;
   paths.infoPlist = paths.infoPlist.replace('<APP_NAME>', appName);
   await setAndroidApplicationVersion(versionText);
-  await setIosApplicationVersion(versionText);
+  // await setIosApplicationVersion(versionText);
   display('');
 };
 
